src/database/models/Warnings.ts

```diff
@@ -0,0 +1,33 @@
+import { Document, model, Schema } from 'mongoose';
+
+export interface Warnings extends Document {
+  serverId: string;
+  messageId: number;
+  channelId: number;
+  notificationId: number;
+}
+
+const WarningsSchema = new Schema({
+  serverId: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  messageId: {
+    type: Number,
+    required: true,
+    unique: true,
+  },
+  channelId: {
+    type: Number,
+    required: true,
+    unique: true,
+  },
+  notificationId: {
+    type: Number,
+    required: true,
+    unique: true,
+  },
+});
```

Comment:
```suggestion
export interface Warnings extends Document {
  serverId: string;
  messageId: string;
  channelId: string;
  notificationId: string;
}

const WarningsSchema = new Schema({
  serverId: {
    type: String,
    required: true,
    unique: true,
  },
  messageId: {
    type: String,
    required: true,
    unique: true,
  },
  channelId: {
    type: String,
    required: true,
    unique: true,
  },
  notificationId: {
    type: String,
    required: true,
    unique: true,
  },
});
```

Discord.js parses IDs as strings, not numbers. This should resolve your fetch errors.

Reply:
Awesome! Thank you!